fj-extension/installation.md

```diff
@@ -10,21 +10,42 @@ previous_url: /get-started/installation
 
 # Managing the Chrome Extension after Installation
 
-After the successful installation of the Fiddler Jam Chrome Extension, you can set its site access and allow it to work in the Incognito mode.
 
-To access the specific settings on the management page of the Fiddler Jam Chrome Extension:
+## Extension Login
 
-1. Click the **Extensions** button on the Google Chrome toolbar next to your profile avatar.
+The Fiddler Jam extension can be used entirely anonymously without the explicit need for a user account and authentication. However, using a single account for the Fiddler Jam extension and Fiddler Jam portal provides the advantage of efficiently managing your previously captured logs through the [**My Captures** section](#my-captures). Login into the Fiddler Jam extension as follows:
 
-1. In the expanded menu, click **Manage Extensions**.
+- Install and start the Fiddler Jam extension.
+- Click on the account icon on the top-right corner of the extension.
+- Login with your Fiddler Jam account.
+
+Note that some of the functionalities like a quick link to the log to the Fiddler Jam portal are available only for [active portal users]({%slug fj-users%}#portal-users).
+
+## My Captures
```

Comment:
My captures will also work for logs recorded on the same machine (if you haven't reinstalled the extension) even if you don't login into the extension.